features/attached_enable.feature

```diff
@@ -59,6 +59,16 @@ Feature: Enable command behaviour when attached to an UA subscription
             """
             This command must be run as root (try using sudo).
             """
+        And I verify that running `ua enable foobar --format json` `as non-root` exits `1`
+        And stdout is formatted as `json` and has keys:
+            """
+            result processed_services failed_services needs_reboot
+            errors warnings
+            """
```

Comment:
Now might be a good opportunity to start using json schemas in our integration tests instead of checking top-level keys. 
What do you think?

Reply:
I think that's a great idea, but maybe we should do this in a separate PR, as I think this one is already too long.
My rationale that this will probably require a new dependency for the json schema while also figuring out how we will define the schemas for the json output that we have and how we will address then on the integration test. But maybe it is way simpler than that, so let me know what you think

Reply:
It should be a separate commit, but up to you on whether it can be this PR or a later one. I do think that this set of features (adding json output to many of our commands) is the right time to do it though.

I don't think it will look too complicated.

We can just pull in https://pypi.org/project/jsonschema/ for the integration test dependencies.

Then we write a schema file in `features/schemas/enable.json`

Then we write a new `step` like `And stdout is json matching the "enable" schema`. That looks like (simplified):
```python
validate(
    instance=json.load(context.stdout),
    schema=json.load(open("./features/schemas/{}.json".format(schema_name)).read())
)
```

Reply:
Ohhh fair enough, I though you also meant performing that validation on the code itself.
But you are right, we can start small and just add it to the test. I will do that

Reply:
oh yeah good point I'm not sure if its worth it to do a schema check at runtime. But I think we can gain a lot from adding it to the integration tests.